react/features/video-menu/components/AbstractMuteButton.js

```diff
@@ -61,7 +59,7 @@ export default class AbstractMuteButton extends AbstractButton<Props, *> {
                 'participant_id': participantID
             }));
 
-        dispatch(openDialog(MuteRemoteParticipantDialog, { participantID }));
+        dispatch(muteRemote(participantID, MEDIA_TYPE.AUDIO));
```

Comment:
So we no longer want to show the warning about muting a participant?

Reply:
No, we only show it for muting multiple participants